demo/src/app/components/modal/demos/customclass/modal-customclass.ts

```diff
@@ -0,0 +1,33 @@
+import {Component, ViewEncapsulation} from '@angular/core';
+
+import {NgbModal, ModalDismissReasons} from '@ng-bootstrap/ng-bootstrap';
+
+@Component({
+  selector: 'ngbd-modal-customclass',
+  templateUrl: './modal-customclass.html',
+  encapsulation: ViewEncapsulation.None,
```

Comment:
I don't love disabling view encapsulation on this, but I wanted to keep the associated styles close to the component. Modals are tough in that (by design) they render content outside of the component's view hierarchy.